### PR TITLE
docs: who supports Ibis

### LINKED ISSUE #7743: docs: Unclear who is behind Ibis from the docs
### Please describe the issue

The docs don't provide much information about how well Ibis is supported and what is the long-term development roadmap. This information would be helpful when considering whether or not to use Ibis as a dependency in other software systems.

For instance, it would be useful to know:

- Does Ibis have any full-time developers or maintainers?
- Are there businesses commited to the development and support of Ibis?
- Can we expect long term support for Ibis?
- Is the API stable, or should we expect major changes or refactoring in the future?
- Are there known issues that should be considered when considering to adopt Ibis as a dependency of a software system (e.g., known incompatibilities)?
- Are there consulting companies that provide enterprise support for Ibis?
- Does Ibis follow well-known security standards?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by lostmygithubaccount**
## Description of changes

add a conceptual doc explaining who supports the Ibis project, based on discussion in the linked issue

## Issues closed

closes #7743

cc: @OlivierBinette

### CODE REVIEW

**Review by cpcloud — APPROVED** [review]

File: docs/concepts/who.qmd [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- [Microsoft's Magpie project](https://www.microsoft.com/en-us/research/project/magpie-2/), built on top of Ibis
+- [SuperDuperDB](https://github.com/SuperDuperDB/superduperdb), bringing AI to any backend Ibis supports
+
+## History
+
+Ibis was originally created by [Wes McKinney](https://wesmckinney.com/). Wes
+created pandas, co-created Apache Arrrow, and co-founded Voltron Data (among
+other things). Ibis was initially a pandas-like dataframe library for Apache
+Impala, but has since grown to support many other backends and mature under the
+stewardship of [Phillip Cloud](https://github.com/cpcloud) and others on the
```
Comment (lostmygithubaccount) [inline docs/concepts/who.qmd:32]: @cpcloud would you want this linking somewhere else? I don't think you have a personal website and linking to youtube seemed weird

Comment (cpcloud) [reply docs/concepts/who.qmd:32]: GitHub is fine, thanks!

**Comment by cpcloud** [comment]
Refer to the `qmd` file directly by path `../tutorials/getting_started.qmd`

**Comment by cpcloud** [comment]
Let quarto worry about what the refs resolve to, and netlify takes care of prettifying links.

**Pull request merged**